app/src/main/java/com/example/tmdb/data/Mapper.kt

```diff
@@ -0,0 +1,14 @@
+package com.example.tmdb.data
+
+import com.example.tmdb.ui.screens.shared.components.MovieItemViewState
+
+fun toMovieItemViewState(
+    movieItem: MovieItem,
+    isFavorite: Boolean
+): MovieItemViewState = MovieItemViewState(
```

Comment:
Instead of passing in `MovieItem`, you could make this an extension function if you want.